ui/components/app/toast-master/selectors.ts

```diff
@@ -107,3 +110,15 @@ export function selectShowConnectAccountToast(
 export function selectSwitchedNetworkNeverShowMessage(state: State): boolean {
   return Boolean(state.metamask.switchedNetworkNeverShowMessage);
 }
+
+/**
+ * Retrieves user preference to see the "New SRP Added" toast
+ *
+ * @param state - Redux state object.
+ * @returns Boolean preference value
+ */
+///: BEGIN:ONLY_INCLUDE_IF(multi-srp)
+export function selectNewSRPAdded(state: State): boolean {
```

Comment:
Since you've changed the `SRP` to `Srp` on the state, we could/should also rename this one too.